### ISSUE TITLE
npm install failure when setup server

### ISSUE THREAD

**Issue by Arktische** [BetaCatPro/RN-XiaoHongShu#1]
It throws the following error when setup server.
```bash
npm ERR! network request to https://verdaccio.sobeter.com/zlogger/-/zlogger-1.1.0.tgz failed, reason: Client network socket disconnected before secure TLS connection was established
```
And I check the accessibility of this URL `https://verdaccio.sobeter.com/zlogger/-/zlogger-1.1.0.tgz`, It failed as expected. No matter how I switch the proxy server, it still failed.

**Comment by llag9810** [comment]
I have encountered the same issue as you have.
The problem lies within the 'resolve' field in the package-lock.json file. If this field exists, npm will fetch the library from the specified URL in this field. I have noticed that the URL 'verdaccio.sobeter.com', which is used in the 'resolve' field, appears to have stopped its service.
As a solution, I delete all the 'resolve' fields and then re-run 'npm install'. This appears to have resolved the issue effectively.
